src/components/WebexMember/WebexMember.jsx

```diff
@@ -22,23 +23,32 @@ export default function WebexMember({
   personID,
   displayStatus,
 }) {
-  const {firstName, lastName} = usePerson(personID);
+  const {firstName, lastName, orgID} = usePerson(personID);
   const me = useMe();
   const members = useMembers(destinationID, destinationType);
   const member = members
     .find((itemMember) => itemMember.ID === personID);
+  const organization = useOrganization(orgID);
```

Comment:
Seems like there are some changes from external user story mingled in here?
Can you make sure the changes only relate to "host"?

Reply:
Done.